dwave/embedding/polynomialembedder.py

```diff
@@ -14,6 +14,10 @@
 #
 # ================================================================================================
 """
+
+**This module is currently under development. It is subject to change and not
```

Comment:
This is inaccurate. It is not under active development, it is simply used as a private source file.

Reply:
Changed in c42bc0c